source/MaterialXCore/Types.h

```diff
@@ -583,6 +583,16 @@ class Matrix44 : public MatrixN<Matrix44, float, 4>
                 m30, m31, m32, m33};
     }
 
+    Vector4 multiply(const Vector4& rhs) const
```

Comment:
Need to investigate why I get somewhat unrelated compile errors when calling this operator* instead of multiply.